tests/route/test_route_flap.py

```diff
@@ -137,6 +139,14 @@ def test_route_flap(duthost, tbinfo, ptfhost, ptfadapter,
     ptf_ip = tbinfo['ptf_ip']
     common_config = tbinfo['topo']['properties']['configuration_properties'].get('common', {})
     nexthop = common_config.get('nhipv4', NHIPV4)
+
+    # On dual-tor, unicast upstream l3 packet destination mac should be vlan mac
+    # After routing, output packet source mac will be replaced with port-channel mac (same as dut_mac)
+    # On dual-tor, vlan mac is different with dut_mac mac. U0/L0 use same vlan mac for AR response
+    # On single tor, vlan mac is same as dut_mac
+    vlan_mac = tbinfo['topo']['properties']['topology']['DUT']['vlan_configs']['one_vlan_a']['Vlan1000']['mac']
```

Comment:
@jcaiMR there is discussion, vlan_mac is only configured for dual tor, for the single tor, this sentence will crashed, change it to try catch or get the vlan_mac when it is dual tor, but keep it as dut_mac for single tor?

Reply:
If vlan config is not one_vlan_a, for example, it could be two_vlan_a  or four_vlan_a. It will throw exception as well. I think hardcode here is not a good idea.

Reply:
resolved, test code will find the first mac in default vlan configuration and will use default vlan interface for testing. 
If vlan mac not exists, will use dut_mac.